packages/web-components/src/index.ts

```diff
@@ -1,7 +1,7 @@
 export * from './badge/index.js';
 export * from './counter-badge/index.js';
+export * from './image/index.js';
 export * from './progress-bar/index.js';
 export * from './spinner/index.js';
 export * from './text/index.js';
-
```

Comment:
The empty line here was intentional to keep the components separate from the rest :-)